tests/integration/components/sl-select-test.js

```diff
@@ -0,0 +1,19 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-select', 'Integration | Component | sl select', {
+    integration: true
+});
+
+test( 'for attribute value on label matches id of input', function( assert ) {
+    this.render( hbs`
+        {{sl-select label='test label'}}
+    ` );
+
+    const wrapper = this.$( '>:first-child' );
+
+    assert.equal(
+        wrapper.find( 'label' ).attr( 'for' ),
+        wrapper.find( 'input' ).attr( 'id' )
+    );
+});
```

Comment:
I thought we were not going to create this file as we have no other tests for this component and have no plans for any until https://github.com/softlayer/sl-ember-components/issues/456